src/components/Popup/AddAccountPage.vue

```diff
@@ -16,6 +16,19 @@
         v-model.number="newAccount.period"
         v-bind:disabled="newAccount.type === OTPType.hotp"
       />
+      <label class="combo-label">{{ i18n.digits }}</label>
+      <select v-model="newAccount.digits">
+        <option v-bind:value="6">6</option>
```

Comment:
Why not use `value` directly here, it seems we needn't change options value later.